imap-proto/src/types.rs

```diff
@@ -12,7 +12,7 @@ impl Copy for AttrMacro {}
 
 #[derive(Debug, Eq, PartialEq)]
 pub enum Response<'a> {
-    Capabilities(Vec<&'a str>),
+    Capabilities(Vec<Capability<'a>>),
```

Comment:
Please add a minor version bump as a separate commit to make sure we semver correctly.